zwave_js_server/model/node.py

```diff
@@ -504,6 +504,9 @@ def handle_ready(self, event: Event) -> None:
         """Process a node ready event."""
         # the event contains a full dump of the node
         self.data.update(event.data["nodeState"])
+        # update device config
+        if new_device_config := self.data.get("deviceConfig"):
```

Comment:
Not sure this is necessary, could just re-assign it regardless, like in the constructor.